Doc/library/os.rst

```diff
@@ -301,7 +301,7 @@ process and user.
 
    .. note::
 
-      On Mac OS X, :func:`getgroups` behavior differs somewhat from
+      On macOS, :func:`getgroups` behavior differs somewhat from
```

Comment:
I have doubts about this one. If the note refers to 10.5 and earlier, it should be Mac OS X.